server/pom.xml

```diff
@@ -52,6 +52,7 @@
   </dependencies>
 
   <build>
+    <testSourceDirectory>src/test/</testSourceDirectory>
```

Comment:
Why is this needed even once https://github.com/diffblue/platform/pull/1974 would be merged? Is it because the project has no `src` folder so even with the fix the path would not be matched?

Reply:
The default is `src/test/java`, but there's no such directory for the test, basically `java` is replaced with something else.